riscv/insns/c_flwsp.h

```diff
@@ -1,7 +1,13 @@
 if (xlen == 32) {
-  require_extension(EXT_ZCF);
-  require_fp;
-  WRITE_FRD(f32(MMU.load<uint32_t>(RVC_SP + insn.rvc_lwsp_imm())));
+  require_either_extension(EXT_ZCF, EXT_ZCMLSD);
+  if (p->extension_enabled(EXT_ZCF)) {
+    require_fp;
+    WRITE_FRD(f32(MMU.load<uint32_t>(RVC_SP + insn.rvc_lwsp_imm())));
+  } else { // c.ldsp - Load pair RV32
+    require_extension(EXT_ZCMLSD);
+    require(insn.rvc_rd() != 0);
```

Comment:
This is redundant with the `require` inside of `WRITE_RD_PAIR`.  It's best to handle this in one place, so delete this one.

Reply:
done in 02fbb0d